cypress/CYPRESS.md

```diff
@@ -158,15 +163,16 @@ End-to-End tests for our project are located in the `cypress/e2e` directory. The
    - Include usage examples and explanations of the purpose of each command.
 
 2. **Team Collaboration**:
+
    - Keep the team informed about testing strategies and updates.
    - Encourage team members to contribute to the testing process and documentation.
 
 By following these guidelines, our E2E tests will be more robust, reliable, and aligned with the best practices in automated testing.
 
-## AWX Cleanup during tests
+## AWX Cleanup during Tests
 
-Many AWX resources have an organization and if the organization is deleted the resource gets deleted. By creating an organization at the start of the tests and using it with resources and then deleting the organization, most resources clean up.
+Many AWX resources have an organization, and if the organization is deleted, the resource gets deleted. By creating an organization at the start of the tests and using it with resources, then deleting the organization, most resources clean up.
 
-There are a few cases where resources do not get cleaned up, such as when a project is syncing, deleting the organization leaves the project around without an associated organization.
+There are a few cases where resources do not get cleaned up, such as when a project is syncing. Deleting the organization leaves the project around without an associated organization.
 
-To handle this, the tests delete E2E organizations older than 2 hours old. Then the other tests delete resources that do not have an organization. This makes the system eventually cleanup the E2E resources.
+To handle this, the tests delete E2E organizations older than 2 hours. Then the other tests delete resources that do not have an organization. This makes the system eventually clean up the E2E resources.
```

Comment:
I do think this statement is true anymore, this one should be revisited.

Reply:
not sure, I wasn't aware of this
@vidyanambiar can help us with this

Reply:
@vishsanghishetty if that is no longer true, we can add a follow up commit.

Reply:
sounds good ✅